swag.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+order: 0
+section: "2018"
+title: "Swag"
+titleDisplay: "Swag"
+parent: "Conference"
+redirect_from:
+  - "/2018/swag/"
+---
+
+What better way to celebrate Doing It With Others than by buying [t-shirts](#t-shirts), [totes](#totes), buttons and stickers?
+
+<br />
+
+## T-Shirts - $20
+<a id="t-shirts"></a>
+
+We have 4 shirt designs available for purchase during [registration](/conference/#registration) (also for sale at the conference). Available S-XL in straight cut and fitted options. Printed locally on Bella+Canvas [3001](https://www.bellacanvas.com/product/3001/Unisex-Jersey-Short-Sleeve-Tee.html) or [6004](https://www.bellacanvas.com/product/6004/Womens-The-Favorite-Tee.html).
+
+<div class="row">
+  <div class="four columns">
+    <h3>2018 T-Shirt Design #1</h3>
+    <p>Limited-run 2018 edition print in "accent"-uated mint green.<br />
+    Printed front and back with shoutouts to our wonderful sponsors.
+    </p>
+  </div>
+  <div class="eight columns">
+    <img src="/images/swag/3-shirt-design.png" alt="Shirt Design #3 detail">
+  </div>
+</div>
+<hr>
+<div class="row">
+  <div class="four columns">
+    <h3>2018 T-Shirt Design #2</h3>
+    <p>Limited-run 2018 edition print in wardrobe-basic black.<br />
+    Printed front and back with shoutouts to our wonderful sponsors.
+    </p>
+  </div>
+  <div class="eight columns">
+    <img src="/images/swag/4-shirt-design.png" alt="Shirt Design #4 detail">
+  </div>
+</div>
+<hr>
+<div class="row">
+  <div class="four columns">
+    <h3>T-Shirt Design Black</h3>
+    <p>Rep Our Networks as timeless in "is-there-any-other-colour?" black.  <br />
+    Printed front and back.
+    </p>
+  </div>
+  <div class="eight columns">
+    <img src="/images/swag/1-shirt-design.png" alt="Shirt Design #1 detail">
+  </div>
+</div>
+<hr>
+<div class="row">
+  <div class="four columns">
+    <h3>T-Shirt Design White</h3>
+    <p>Rep Our Networks as timeless in white, but keep it monochome if you must.<br />
+    Printed front and back.
+    </p>
+  </div>
+  <div class="eight columns">
+    <img src="/images/swag/2-shirt-design.png" alt="Shirt Design #2 detail">
+  </div>
+</div>
+
+<br />
+
+## Totes - $15
```

Comment:
I was expecting single-sided print, let me confirm if we can still offer this at $15.

Reply:
From signal convo we seem good to go ahead!